logging/troubleshooting/cluster-logging-cluster-status.adoc

```diff
@@ -1,18 +1,12 @@
 :_mod-docs-content-type: ASSEMBLY
 :context: cluster-logging-cluster-status
 [id="cluster-logging-cluster-status"]
-= Viewing OpenShift Logging status
+= Viewing Logging status
```

Comment:
Yeah, but I was updating it to be consistent with what's in the left nav for the moment. I think it would be a follow up task to update it across the board for the logging docs because pretty much all the topics use "Logging" in the left nav currently.

Reply:
Ah yeah totally makes sense